Makefile

```diff
@@ -114,6 +114,10 @@ format: .bin/goimports
 docker:
 		docker build -f .docker/Dockerfile-build -t oryd/kratos:latest-sqlite .
 
+# Runs the documentation tests
+test-docs:
+		text-run README.md
```

Comment:
If this is instead `npm run text-run` and `text-run` is a package.json run script, then this should already work in the CI!

Reply:
Thanks for the suggestions! I implemented all of them but it doesn't work on CI.

- When I run this inside the `test` job (as suggested by you), it doesn't find `npm`. I assume this job runs inside a Go container. 
- When I run this inside the `validate` job, it doesn't have the Kratos binary.

I think the `validate` job seems a good place to run this. Do you have any pointers how to get the Kratos binary into that container in a way that is idiomatic with your setup?

Reply:
Ah ok, I should have seen that when writing my comment. The default go image does not have NodeJS, but [there is one which does](https://circleci.com/developer/images/image/cimg/go):

```
    docker:
      - image: cimg/go:1.15.6
```

Replacing https://github.com/ory/kratos/pull/567/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R16 with that image should then make this work!

Reply:
Thanks for the pointer! For future reference, the `cimg/go:1.15-node` image has a folder structure that is incompatible with the folder structure CircleCI expects. This causes the code checkout step to fail. The `circleci/golang:1.15-node` image doesn't have this problem and seems to work well.